sentry_sdk/integrations/starlette.py

```diff
@@ -339,6 +338,12 @@ async def form(self):
         curl -X POST http://localhost:8000/upload/somethign -H "Content-Type: application/x-www-form-urlencoded" -d "username=kevin&password=welcome123"
         curl -X POST http://localhost:8000/upload/somethign  -F username=Julian -F password=hello123
         """
+        try:
+            # Optional dependency of Starlette to parse form data.
+            import multipart  # type: ignore # noqa: F401
```

Comment:
I'd do at the top of the file
```python
try:
    import multipart
except ImportError:
    multipart = None
```

and then
```
if multipart is None:
    return None
```
in the function

Reply:
nice!